packages/liveblocks-zustand/src/index.test.ts

```diff
@@ -279,17 +275,15 @@ describe("middleware", () => {
 
       socket.callbacks.open[0]!();
 
-      expect(socket.sentMessages[0]).toEqual(
-        JSON.stringify([
-          {
-            type: ClientMsgCode.UPDATE_PRESENCE,
-            data: { cursor: { x: 0, y: 0 } },
-          },
-          {
-            type: ClientMsgCode.FETCH_STORAGE,
-          },
-        ])
-      );
+      expect(JSON.parse(socket.sentMessages[0])).toEqual([
+        {
+          type: ClientMsgCode.UPDATE_PRESENCE,
+          data: { cursor: { x: 0, y: 0 } },
+        },
+        {
+          type: ClientMsgCode.FETCH_STORAGE,
+        },
+      ]);
```

Comment:
Sorry about the prettier-induced noise here. The structural change made here is:

```expect(xxx).toEqual(JSON.stringify(yyy))```

→

```
expect(JSON.parse(xxx!)).toEqual(yyy)
```

So that these tests are more robust in the light of field ordering.